collector/go.mod

```diff
@@ -16,12 +16,12 @@ replace cloud.google.com/go => cloud.google.com/go v0.107.0
 
 require (
 	github.com/golang-collections/go-datastructures v0.0.0-20150211160725-59788d5eb259
-	github.com/open-telemetry/opentelemetry-collector-contrib/confmap/provider/s3provider v0.71.0
-	github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents v0.71.0
-	github.com/stretchr/testify v1.8.1
-	go.opentelemetry.io/collector v0.71.0
-	go.opentelemetry.io/collector/component v0.71.0
-	go.opentelemetry.io/collector/confmap v0.71.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/confmap/provider/s3provider v0.72.0
+	github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents v0.72.0
+	github.com/stretchr/testify v1.8.2
+	go.opentelemetry.io/collector v0.72.0
```

Comment:
collector v0.73.0 was released yesterday. Is it worth moving to 0.73.0 directly?